diffrax/_solver/dopri5.py

```diff
@@ -90,9 +90,9 @@ class Dopri5(AbstractERK):
     """
 
     tableau: ClassVar[ButcherTableau] = _dopri5_tableau
-    interpolation_cls: ClassVar[
-        Callable[..., _Dopri5Interpolation]
-    ] = _Dopri5Interpolation
+    interpolation_cls: ClassVar[Callable[..., _Dopri5Interpolation]] = (
+        _Dopri5Interpolation
+    )
```

Comment:
Can we skip the spurious reformatting?

(Let's put that in a separate PR if you like? Perhaps this is coming from bumping `ruff`.)

Reply:
Yes, I'll drop it, sorry.